build.sbt

```diff
@@ -1,6 +1,8 @@
 name := "distr_stacktrace"
 
-version := "1.0"
+organization := "org.mariusz89016"
```

Comment:
Don't worry about this now, it'll be corrected during my task. But yes, these are good remarks.